CEDS/MSSQL/0015-View-SchoolYearsDim-Create.sql

```diff
@@ -0,0 +1,24 @@
+-- SPDX-License-Identifier: Apache-2.0
+-- Licensed to the Ed-Fi Alliance under one or more agreements.
+-- The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+-- See the LICENSE and NOTICES files in the project root for more information.
+IF EXISTS (
+        SELECT 1
+        FROM INFORMATION_SCHEMA.VIEWS
+        WHERE TABLE_SCHEMA = 'analytics'
+            AND TABLE_NAME = 'ceds_SchoolYearsDim'
+        )
+BEGIN
+    DROP VIEW analytics.ceds_SchoolYearsDim;
+END;
+GO
+
+CREATE OR ALTER VIEW analytics.ceds_SchoolYearsDim
+AS (
+    SELECT
+        Id AS DimSchoolYearId,
+        SchoolYear,
+        CONCAT('01-07-',SUBSTRING(SchoolYearDescription, 1,4)) AS SessionBeginDate,
+        CONCAT('06-30-',SUBSTRING(SchoolYearDescription, 6,9)) AS SessionEndDate
+    FROM edfi.SchoolYearType
```

Comment:
We are missing the `LastModifiedDate` field. In this case we don't have to find the max value, since it's just one table we are getting all the data from.

Reply:
Working on it.